src/Utilities.UnitTests/FlowAnalysis/Analysis/PropertySetAnalysis/PropertySetAnalysisTests.cs

```diff
@@ -1270,5 +1272,160 @@ public PropertySetAnalysisTests(ITestOutputHelper output)
         {
             this.TestOutput = output;
         }
+
```

Comment:
All the following lines are copy-pasted from `DiagnosticAnalyzerTestBase` as I couldn't find a way to achieve the specific checks that are done in this class.

@sharwell If you have any good idea, please feel free to share and I'll take care to do the update.

Reply:
@Evangelink Can you please retarget the changes to this file 2.9.x branch? `PropertySetAnalysisTests` are used by security analyzers, and will likely be updated in 2.9.x branch more often then in master.